src/test/scenarios/loadPlugins.test.js

```diff
@@ -133,6 +133,163 @@ describe("load plugins", function () {
     PluginActions.requestPlugins();
   });
 
+  it("handles missing modules metadata in plugin request response gracefully",
+    function (done) {
+      PluginActions.request = ajaxWrapperStub((url, resolve) => {
+        resolve({
+          "body": {
+            "plugins": [
+              {
+                "id": "plugin-id",
+                "implementation": "package.class",
+                "info": {
+                  "description": "Plugin description.",
+                  "name": "Plugin Name"
+                },
+                "plugin": "mesosphere.marathon.plugin.http.HttpRequestHandler",
+                "tags": [
+                  "ui",
+                  "example"
+                ]
+              }
+            ]
+          }
+        });
```

Comment:
Can we have a base `plugins` array that we can then `extend()` by adding the test-specific properties as needed? We are repeating the same structure 4 times with the exception of `info.modules` - I guess the tests would be more terse that way.